cmd/go-filecoin/log.go

```diff
@@ -38,14 +36,12 @@ Outputs event log messages (not other log messages) as they are generated.
 	},
 
 	Run: func(req *cmds.Request, re cmds.ResponseEmitter, env cmds.Environment) error {
-		r, w := io.Pipe()
+		r := logging.NewPipeReader()
```

Comment:
I think this creates a reader for leveled log messages produced by go-log, right? If so the `ShortDescription` and `Tagline` fileds in the `cmdkit.HelpText` struct above should be updated to state this.

Reply:
Done